videochat/src/config.js

```diff
@@ -28,38 +28,38 @@ export const appConfig = {
     //   statsReportTimeInterval: 5
   },
   endpoints: {
-    api: "api.connectycube.com",
-    chat: "chat.connectycube.com"
+    api: "REPLACE_APP_API_DOMAIN",
```

Comment:
for a sake of simplicity, let's remove endpoints configuration code
Shared users do not need this to change